src/components/ToDo/ToDo.tsx

```diff
@@ -0,0 +1,127 @@
+import classNames from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo
+  isTemp?: boolean;
+  onRemove?: (todoId: number) => void
+  isEditing?: boolean
+  onStatusChange?: (todoId: number, data: boolean) => void;
+  onTitleChange?: (todoId: number, title: string) => void;
+};
+
+export const ToDo: React.FC<Props> = ({
+  todo,
+  isTemp = false,
+  onRemove = () => { },
+  isEditing,
+  onStatusChange = () => { },
+  onTitleChange = () => { },
```

Comment:
why are all functions not required? You use `ToDo` component once (in `ToDoList`) and pass all of them (and all are always defined)

Reply:
`isTemp` and `isEditing` are always defined as well

Reply:
seems like it is remaining from the previous version of the implementation.
 Didn't notice that I haven't removed the optionality